record-and-playback/core/scripts/archive/archive.rb

```diff
@@ -39,6 +39,16 @@ def archive_events(meeting_id, redis_host, redis_port, raw_archive_dir, break_ti
   #end
 end
 
+def archive_note(meeting_id, note_endpoint, note_formats, raw_archive_dir)
+  BigBlueButton.logger.info("Archiving notes for #{meeting_id}")
+  note_dir = "#{raw_archive_dir}/#{meeting_id}/note"
+  FileUtils.mkdir_p(note_dir)
+  note_id = FNV.new.fnv1a_32(meeting_id).to_s(16)
+  note_formats.each do |format|
+    BigBlueButton.try_download("#{note_endpoint}/#{note_id}/export/#{format}", "#{note_dir}/note.#{format}")
+  end
+end
```

Comment:
We should delete the note from etherpad after archiving it, so it's not accumulating them endlessly. (Make sure to only delete it if `break_timestamp` is nil, since otherwise the meeting might be ongoing.)

Reply:
There's already a plugin on Etherpad to cleanup inactive notes, I'll check if the current build of bbb-etherpad is configuring it properly.